frontend/awx/resources/credentials/CredentialForm.tsx

```diff
@@ -22,19 +23,21 @@ import { getAwxError } from '../../useAwxView';
 export function CreateCredential() {
   const { t } = useTranslation();
   const navigate = useNavigate();
-  const onSubmit: PageFormSubmitHandler<Credential> = async (editedCredential, setError) => {
+  const onSubmit: PageFormSubmitHandler<Credential> = async (credential, setError) => {
     try {
-      try {
-        const organization = await getOrganizationByName(
-          editedCredential.summary_fields.organization.name
-        );
-        if (!organization) throw new Error(t('Organization not found.'));
-        editedCredential.organization = organization.id;
-      } catch {
-        throw new Error(t('Organization not found.'));
+      if (credential.summary_fields.organization.name) {
```

Comment:
If the org is not provided we should provide the current user in the payload:
https://github.com/ansible/awx/blob/devel/awx/ui/src/screens/Credential/CredentialAdd/CredentialAdd.js#L61

Reply:
done